front/app/containers/LandingPage/LandingPage.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from 'utils/testUtils/rtl';
+import LandingPage from '.';
+
+jest.mock('services/locale');
+jest.mock('services/appConfiguration');
+jest.mock('utils/analytics');
+jest.mock('utils/cl-router/withRouter');
+jest.mock('utils/cl-router/Link');
+
+jest.mock('services/homepageSettings', () => ({
+  updateHomepageSettings: jest.fn(),
+}));
+
+const mockHomepageSettings = {
+  data: {
+    id: '1',
+    attributes: {
+      banner_layout: 'full_width_banner_layout',
+      banner_signed_out_header_multiloc: { en: 'Signed out header' },
+      banner_signed_out_subheader_multiloc: { en: 'Signed out subhead' },
+      banner_signed_in_header_multiloc: { en: 'Signed in header' },
+      header_bg: { large: 'https://example.com/image.png' },
+    },
+  },
+};
+
+jest.mock('hooks/useHomepageSettings', () => {
+  return jest.fn(() => mockHomepageSettings);
+});
+
+describe('<LandingPage />', () => {
+  it('renders with HomepageSettings for logged out users', () => {
```

Comment:
Should this include a bit more tests - like are there different scenarios for the different combinations of the homepage settings? Should we check it renders what we expect in the different scenarios?

Reply:
I ended up covering all of the test cases I thought were relevant with the e2e ones, do you think it makes sense to also cover them here?

Reply:
Perhaps not, if it's the exact same tests. But normally unit tests would be best suited for the components themselves and e2e tests are for the entire user flow as a whole. Do we have some new/ changed components that handle some more complex / bug prone logic and could benefit from a unit test?